index.html

```diff
@@ -1,657 +1,1131 @@
+                                  <div class="button join-a-group">
+                                      <a href="#" class="cmp-button" type="button">
+                                          <span class="cmp-button__text">Join A Group</span>
+                                      </a>
+                                  </div>
+                                  <div class="button start-a-group">
+                                      <a href="#" class="cmp-button" type="button">
+                                          <span class="cmp-button__text">Start A Group</span>
+                                      </a>
+                                  </div>
+                              </div>
+                          </div>
+                      </div>
+                  </div>
+              </div>
+          </div>
+        </div>
+
+        
+        <div class="container responsivegrid slider featured-products aem-GridColumn aem-GridColumn--default--12">
+          <div class="cmp-container">
+              <div class="aem-Grid aem-Grid--12 aem-Grid--default--12">
+                  <div class="responsivegrid aem-GridColumn product-title aem-GridColumn--default--12 layout-grid">
+                      <div class="cmp-container">
+                          <div class="title aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                              <div id="title-bcb754f8400" class="cmp-title">
+                                  <h5 class="cmp-title__text">Featured Products</h5>
+                              </div>
+                          </div>
+                          <div class="button btn-link aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <a id="button-0aa668f273" class="cmp-button" href="https://www.amazon.com/s?k=Leaderimpact&amp;i=digital-text&amp;dc&amp;qid=1591796773&amp;ref=sr_nr_i_2">
+                              <span class="cmp-button__text">Shop All</span>
                             </a>
-                        </div>
-                        <div class="button">
-                            <a href="" id="" class="btn-link" data-cmp-clickable="" data-cmp-data-layer="">
-                                <span class="btn-text">Join a Group</span>
+                          </div>
+                      </div>
+                  </div>
+                  <div class=" responsivegrid product-slider-wrap aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                      <div class="cmp-container">
+                          <div class="container responsivegrid product-item aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                              <div class="cmp-container">
+                                  <div class="teaser aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                      <div class="cmp-teaser">
+                                          <div class="cmp-teaser__image post-product__image">

```

Comment:
@Prystal254 We can only add custom classes to the root element of the component. `.post-product__image` can't go here.